src/isomorphic/children/__tests__/ReactChildren-test.js

```diff
@@ -431,13 +429,14 @@ describe('ReactChildren', function() {
     expect(console.error.calls[0].args[0]).toContain('use of a keyed object');
   });
 
+  /*eslint-disable no-unused-expressions */
```

Comment:
Instead of disabling this rule, maybe we chould use the `void` operator on each place that triggers it (which this rule's docs claim doesn't warn). Like `void frag[key];`.